static/js/drafts.js

```diff
@@ -106,6 +106,24 @@ export const draft_model = (function () {
     return exports;
 })();
 
+// A one-time fix for buggy drafts that had their topics
+// renamed to `undefined` in #23238. Exported for testing.
+// TODO/compatibility: This can be deleted in 2023 since
+// all relevant drafts will have either been run through
+// this code or else been deleted after 30 (DRAFT_LIFETIME) days.
+let fixed_buggy_drafts = false;
+export function fix_drafts_with_undefined_topics() {
+    const data = draft_model.get();
+    for (const draft_id of Object.keys(data)) {
```

Comment:
Won't the `topic` be undefined for any draft private message?

Reply:
Ah yes thanks, I will add a check for it being a stream message as well.